### PR TITLE
Javascript/javascript1/week4

### PR DESCRIPTION
**Pull request by islam-fawzy25**

### CODE REVIEW

File: javascript/javascript1/week4/voiceAssistant.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+ const nameArr=[];
+ const myTodo =[];
+
+  function getReply(command){ 
+    const splitCommand = command.split(' ');
+    const userName = splitCommand[4];
+    const myTodoIs =myTodo.join();
+
+      // if statment for checking and resturn the user name :
+      if(command.startsWith('Hello my name is')){
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/voiceAssistant.js:12]: What if `command = "hello my name is"`? Use `command.toLowerCase();` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase

File: javascript/javascript1/week4/voiceAssistant.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      if(command.startsWith('Remove') && command.endsWith('todo')){
+        splitCommand.splice(0,1);
+        splitCommand.splice(splitCommand.length-3,3);
+         const removeCommand = splitCommand.join(' ')
+
+         if(myTodo.includes(removeCommand)){
+           return myTodo.splice(myTodo.indexOf(removeCommand),1)
+         }
+        return myTodoIs
+         };
```
Comment (cameronjdavis) [inline javascript/javascript1/week4/voiceAssistant.js:45]: Use consistent code formatting. Search for "visual studio code format document" to see how Visual Studio Code can format your code for you.

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/voiceAssistant.js (hunk 1 above)
Comment (islam-fawzy25) [reply javascript/javascript1/week4/voiceAssistant.js]: yes thanks it is good idea 😃

File: javascript/javascript1/week4/voiceAssistant.js (hunk 2 above)
Comment (islam-fawzy25) [reply javascript/javascript1/week4/voiceAssistant.js]: yes done 😃

**Pull request merged**